File: mobile/lib/core/constants/paises.dart
/// Espejo de frontend/src/utils/paises.ts — países soportados al registrar
/// una empresa nueva (define documento de identidad, moneda y zona horaria
/// por defecto, ver backend/src/common/constants/*-por-pais.ts).
class PaisInfo {
  final String codigo;
  final String nombre;
  const PaisInfo(this.codigo, this.nombre);
}

const List<PaisInfo> paises = [
  PaisInfo('DO', 'República Dominicana'),
  PaisInfo('US', 'Estados Unidos'),
  PaisInfo('MX', 'México'),
  PaisInfo('GT', 'Guatemala'),
  PaisInfo('HN', 'Honduras'),
  PaisInfo('SV', 'El Salvador'),
  PaisInfo('NI', 'Nicaragua'),
  PaisInfo('CR', 'Costa Rica'),
  PaisInfo('PA', 'Panamá'),
  PaisInfo('CO', 'Colombia'),
  PaisInfo('PE', 'Perú'),
  PaisInfo('EC', 'Ecuador'),
  PaisInfo('VE', 'Venezuela'),
  PaisInfo('CL', 'Chile'),
  PaisInfo('AR', 'Argentina'),
  PaisInfo('BR', 'Brasil'),
  PaisInfo('ES', 'España'),
];

File: frontend/src/utils/paises.ts
/** Países soportados al registrar una empresa nueva — define documento de
 * identidad, moneda y zona horaria por defecto (ver documentosIdentidad.ts,
 * backend/monedas-por-pais.ts y zonasHorarias.ts). */
export const PAISES: { codigo: string; nombre: string }[] = [
  { codigo: 'DO', nombre: 'República Dominicana' },
  { codigo: 'US', nombre: 'Estados Unidos' },
  { codigo: 'MX', nombre: 'México' },
  { codigo: 'GT', nombre: 'Guatemala' },
  { codigo: 'HN', nombre: 'Honduras' },
  { codigo: 'SV', nombre: 'El Salvador' },
  { codigo: 'NI', nombre: 'Nicaragua' },
  { codigo: 'CR', nombre: 'Costa Rica' },
  { codigo: 'PA', nombre: 'Panamá' },
  { codigo: 'CO', nombre: 'Colombia' },
  { codigo: 'PE', nombre: 'Perú' },
  { codigo: 'EC', nombre: 'Ecuador' },
  { codigo: 'VE', nombre: 'Venezuela' },
  { codigo: 'CL', nombre: 'Chile' },
  { codigo: 'AR', nombre: 'Argentina' },
  { codigo: 'BR', nombre: 'Brasil' },
  { codigo: 'ES', nombre: 'España' },
];
